File: CondorV/CondorV/Models/BD/Mesure.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CondorV.Models.BD
{
    public class Mesure
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public long Id { get; set; }    
        public DateTime Date { get; set; }
        public double Valeur { get; set; }
        public long GrandeurId { get; set; }
      
        public Grandeur? Grandeur { get; set; }

    }
}

File: stageTemplates/src/app/models/mesure.ts
import { Grandeur } from "./grandeur";

export class Mesure {
    id: number;
    date: Date;
    valeur?: number | null;
    grandeurId: number;
    grandeur?: Grandeur | null;
  
    constructor(id: number, date: Date,valeur: number|null, grandeurId: number) {
      this.id = id;
      this.date = date;
      this.valeur=valeur;
      this.grandeurId = grandeurId;
    }
  }
  